### ISSUE TITLE
Post-harmony: Investigate C:SL "new features" panel

### ISSUE THREAD

**Issue by aubergine10** [krzychu124/Cities-Skylines-Traffic-Manager-President-Edition#420]
We've got some big UI updates plant for the 11.x branch - would be good to introduce those changes to users via the C:SL "new features" panel that pops up when a new DLC is released, assuming we can find how to interact with it.

Alternatively we could just make our own panel.

**Comment by krzychu124** [comment]
I would like to use `stars` effect 😄

**Comment by aubergine10** [comment]
TAM mod used the stars effect, so it is definitely usable from main menu :) Will actually yield much better quality stars as well due to very high framerate at main menu.

**Comment by krzychu124** [comment]
Yeah, we need only reference to camera and gameObject with material because it's shader :)

**Comment by aubergine10** [comment]
We could show stars after successfully completing a timed traffic light sequence 🤣

**Comment by aubergine10** [comment]
Tagging #118
